packages/server/src/notifier/model/event/forum.ts

```diff
@@ -16,6 +16,9 @@ export const fromPostAddedEvent: NotifEventFromQNEvent<'PostAddedEvent'> = async
 
   const eventData = pick(postAddedEvent, 'inBlock', 'id')
 
+  // TODO: add FORUM_POST_REPLY support
+  // TODO: somehow restrict number of notifs per event to avoid spamming
```

Comment:
This is done [here](https://github.com/Joystream/pioneer/blob/backend/packages/server/src/notifier/model/notifications.ts#L67) but it's at the end of the whole aggregation process in order to prioritize based on the user preferences.

Reply:
thanks, missed that